### ISSUE TITLE
DBus's PropertiesChanged is not emitted when a Unit stops successfully

### ISSUE THREAD

**Issue by radium226** [systemd/systemd#3390]
### Submission type
- [x] Bug report
- [ ] Request for enhancement (RFE)
### systemd version the issue has been seen with

> 229
### Used distribution

> Arch Linux
### Step to reproduce

Everything is in this thread: https://lists.freedesktop.org/archives/systemd-devel/2016-May/036619.html :smile: 

I noticed that the DBus event `PropertiesChanged` is not emitted when a Unit stop successfully (but triggered if the Unit stop in a failure state).  

I hope it's enough, but do not hesitate if I need to add more informations.

**Comment by poettering** [comment]
How does your unit look like precisely?

**Comment by radium226** [comment]
I tried with a very basic `fake.service` unit which called a `fake` script which wait 5 seconds before exiting in success:

```
[Unit]
Description=Fake Service

[Service]
Type=simple
ExecStart=/usr/local/bin/fake
```

I also tried with `Type=oneshot`.

Thanks a lot for your help!

**Comment by radium226** [comment]
Is there anything new about that issue? I do not have enough knowledge to go deep in Systemd's code, but I'll be happy to help if needed!

**Comment by poettering** [comment]
Fixed by 0dd99f86addd1f81e24e89807b6bc4aab57d5793, closing
